R/version-utils.R

```diff
@@ -0,0 +1,41 @@
+source("R/constants.R")
+source("R/semver-handling.R")
+#' Get latest version
+#'
+#' Get the latest version from vector of file names
+#'
+#' @param file_names string or string vector containing file names with valid semantic versioning
+#'
+#' @return vector of strings containing latest version and latest version file name
+get_latest_version <- function() {
```

Comment:
Can we change this function to return the dictionary file to use and version, rather then the name of the dictionary file name and version number?

It looks like the parts, sets, and tables qmds don't need the dictionary file name for any reason other than to get the dictionary itself.

Reply:
get_latest_version -> get_dictionary_file